test_missing_imports.py

```
@@ -0,0 +1,302 @@
+#!/usr/bin/env python3
+"""Test the improved error handling for missing imports."""
+
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)